ale/base/base.py

```diff
@@ -323,3 +326,33 @@ def center_ephemeris_time(self):
     @property
     def short_mission_name(self):
         return self.__module__.split('.')[-1].split('_')[0]
+
+    @property 
+    def projection(self):
+        if not hasattr(self, "_projection"): 
+            try: 
+              from osgeo import gdal 
+            except: 
+                self._projection = None
+                return self._projection
```

Comment:
Localized the import to keep the optional gdal stuff in one location without complicating things by adding some gdal module or something.